exercises/concept/intro-select/report-results.sh

```diff
@@ -0,0 +1,15 @@
+DB_FILE=$1
+SLUGS=$(jq -n --slurpfile test_data test_data.json '$test_data[0] | keys' | sed 's/[][",]//g')
+
+# Generate result for each test
+rm -f results.txt
+for SLUG in $SLUGS; do
+    ACTUAL=$(sqlite3 -json $DB_FILE "SELECT * FROM ${SLUG};" | tr -d '[:space:]')
+    if [ -z "$ACTUAL" ]; then
+        ACTUAL="[]"
+    fi
+    jq -n --slurpfile test_data test_data.json --argjson got ''${ACTUAL}'' --arg slug ${SLUG} -f test-result.jq >> results.txt
```

Comment:
Two single quotes are not the same as a double quote.
```suggestion
    jq -n --slurpfile test_data test_data.json --argjson got "${ACTUAL}" --arg slug "${SLUG}" -f test-result.jq >> results.txt
```